edocumentation-domain/src/main/resources/db/migration/V3_0__dcsa_im_v3.sql

```diff
@@ -594,3 +594,11 @@ CREATE TABLE shipment_event (
     document_id uuid NOT NULL,
     reason varchar(5000) NULL
 );
+
+  CREATE TABLE advance_manifest_filing (
+	manifest_id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+    shipment_id uuid NOT NULL REFERENCES shipment(id),
+    manifest_type_code varchar(10) NOT NULL,
```

Comment:
I know that this is also 10 in the API - I am investigating if this should be changed to 50

Reply:
Done.